src/components/DateRangePicker.jsx

```diff
@@ -469,8 +473,9 @@ class DateRangePicker extends React.PureComponent {
 
     const withAnyPortal = withPortal || withFullScreenPortal;
 
+    /* eslint-disable jsx-a11y/no-static-element-interactions, jsx-a11y/click-events-have-key-events */
```

Comment:
Can we move it down to where it originally was and use `eslint-disable-line`

Reply:
done 👌 I was trying to stay within the character limit but the line length is too long either way lol